src/waveform/waveformwidgetfactory.cpp

```diff
@@ -1028,14 +1033,29 @@ void WaveformWidgetFactory::evaluateWidgets() {
             useOpenGl = allshader::SimpleWaveformWidget::useOpenGl();
             useOpenGles = allshader::SimpleWaveformWidget::useOpenGles();
             useOpenGLShaders = allshader::SimpleWaveformWidget::useOpenGLShaders();
-            developerOnly = allshader::SimpleWaveformWidget::developerOnly();
+            category = allshader::SimpleWaveformWidget::category();
+            break;
+#endif
+        case WaveformWidgetType::AllShaderHSVWaveform:
+#ifndef MIXXX_USE_QOPENGL
+            continue;
+#else
+            widgetName = allshader::HSVWaveformWidget::getWaveformWidgetName();
+            useOpenGl = allshader::HSVWaveformWidget::useOpenGl();
+            useOpenGles = allshader::HSVWaveformWidget::useOpenGles();
+            useOpenGLShaders = allshader::HSVWaveformWidget::useOpenGLShaders();
+            category = allshader::HSVWaveformWidget::category();
             break;
 #endif
         default:
             DEBUG_ASSERT(!"Unexpected WaveformWidgetType");
             continue;
         }
 
+        if (category == WaveformWidgetCategory::Legacy) {
+            widgetName += " " + tr("(legacy)");
```

Comment:
lets ensure the translators don't swallow the parentheses.
```suggestion
            widgetName += QStringLiteral(" (%1)").arg(QObject::tr("legacy"));
```